src/main/java/com/cburch/logisim/fpga/hdlgenerator/Hdl.java

```diff
@@ -88,6 +89,25 @@ public static String getLineCommentStart() {
     return isVhdl() ? "-- " : "// ";
   }
 
+  public static String startIf(String condition) {
+    return isVhdl() ? LineBuffer.formatHdl("IF {{1}} THEN", condition)
+                    : LineBuffer.formatHdl("if ({{1}}) begin", condition);
+  }
+
+  public static String elseStatement() {
+    return isVhdl() ? Vhdl.getVhdlKeyword("ELSE") : "end else begin";
+  }
+
+  public static String elseIf(String condition) {
+    return isVhdl() ? LineBuffer.formatHdl("{{1}} {{2}} {{3}}", Vhdl.getVhdlKeyword("ELSIF"), condition, Vhdl.getVhdlKeyword("THEN"))
+                    : LineBuffer.formatHdl("end else if ({{1}}) begin", condition);
+  }
+
+  public static String endIf() {
+    return isVhdl() ? LineBuffer.formatHdl("{{1}} {{2}};", Vhdl.getVhdlKeyword("END"), Vhdl.getVhdlKeyword("IF"))
```

Comment:
I'd not use `LineBuffer` here. It just adds overhead w/o no benefits as formatting string is just its arguments (consts) + space. Simply concatenate with `getVhdlKeyword()` calls with space in between.